index.html

```diff
@@ -26,11 +26,11 @@
         </header>
         <main>
             <h1>Welcome to Alain's CSD120 Website!</h1>
-            <picture>
-                <source srcset="images/150.jpg" media="(max-width:600px)">
-                <source srcset="images/300.jpg">
-                <a href="https://imgur.com/gallery/5d9P7vl"><img src="images/300.jpg" alt="images/300.jpg"></img></a>
-            </picture>
+            <a href="images/original.jpg">
+                <img src="images/300.jpg" alt="A selfie of Alain"
+                    srcset="images/150.jpg 150w, images/300.jpg 300w"
+                    sizes="(max-width: 500px) 150px, 300px">
+            </a>
```

Comment:
No need for a `<picture>` here.  Just use srcset and sizes.
Alt text shouldn't just have the file path of the image.